DataFormats/L1TMuon/src/EMTFHit.cc

```diff
@@ -1,4 +1,28 @@
 #include "DataFormats/L1TMuon/interface/EMTFHit.h"
 
 namespace l1t {
+
+  CSCDetId EMTFHit::CreateCSCDetId() const {
+    return CSCDetId( (endcap == 1) ? 1 : 2, station,    // For now, leave "layer" unfilled, defaults to 0.
+         (ring == 4) ? 1 : ring, chamber ); // Not sure if this is correct, or what "layer" does. - AWB 27.04.16
```

Comment:
@abrinke1 layer should be 0, e.g. https://github.com/cms-sw/cmssw/blob/master/L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.cc#L198-L199.